inc/class-cachify.php

```diff
@@ -514,6 +546,30 @@ public static function robots_txt( $data ) {
 		return $data;
 	}
 
+	/**
+	 * HDD Cache expiration cron action
+	 *
+	 * @since  2.3.3
+	 */
+	public static function run_hdd_cache_cron() {
+		Cachify_HDD::clear_cache();
+	}
+
+	/**
+	 * Add cache expiration cron shedule
+	 *
+	 * @since  2.3.3
+	 *
+	 * @return array
+	 */
+	public static function add_cron_cache_expiration() {
+		$schedules['cashify_cache_expire'] = array(
+			'interval' => self::$options['cache_expires'] * 3600,
+			'display'  => 'Cachify',
```

Comment:
The string should be translatable.

Reply:
I'm not sure if this is really needed, because this is the shedule name, but i changed it in my last commit.